errors.go

```diff
@@ -113,6 +113,76 @@ func Must6[T1, T2, T3, T4, T5, T6 any](val1 T1, val2 T2, val3 T3, val4 T4, val5
 	return val1, val2, val3, val4, val5, val6
 }
 
+// just assume fine if err is nil or true.
+func fine(err any) bool {
```

Comment:
I won't use `err` as a variable name here, as the type is unknown.

I would update the signature of all Just* methods

But it's the way must was written, so …

Reply:
This is a reasonable suggestion,  maybe I can adjust `must` along with it.

Reply:
How do you feel about changing `err` to `failure`?

Reply:
Naming is a pain.

@samber is better than I am 😬😂

Reply:
I have a good friend named ChatGPT. Perfect for brainstorming.